script-library/container-features/src/features.json

```diff
@@ -1065,12 +1065,61 @@
 				"nonFreePackages": {
 					"type":"boolean",
 					"default": true,
-					"description": "Add packages from non-free Debain repository?"
+					"description": "Add packages from non-free Debian repository?"
 				}
 			},
 			"buildArg": "_VSC_INSTALL_COMMON",
-			"include": []
+			"include": [
+				"puppet",
```

Comment:
does the CLI need puppet and chef? I don't see any other references in this PR.

Reply:
This is not part of the dotnet feature, but an artifact of other changes in this file as it is a high-churn file.

See [this comment](https://github.com/microsoft/vscode-dev-containers/pull/1210#issuecomment-995269025) above